sqlserver/tests/test_integration.py

```diff
@@ -568,6 +571,15 @@ def test_resolved_hostname_set(
         for k, v in cloud_metadata.items():
             instance_docker[k] = v
     instance_docker['dbm'] = dbm_enabled
+    instance_docker['procedure_metrics'] = {'enabled': False}
+    instance_docker['query_metrics'] = {
+        'enabled': True,
+        'run_sync': True,
+        'collection_interval': 1,
+        # in tests sometimes things can slow down so we don't want this short deadline causing some events
+        # to fail to be collected on time
```

Comment:
if this is true, do query metrics need to be enabled at all here? Is there something actually being tests on that code path in these integration tests? 

sorry for my ignorance, just don't remember

Reply:
I'm not sure, but this comment is found elsewhere too, so maybe it's just a copy paste.